docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -132,13 +133,13 @@ Property Name                                                 Description
                                                               queries fail with *"not enough replicas"*. The other possible
                                                               values are ``DOWNGRADING_CONSISTENCY`` and ``FALLTHROUGH``.
 
-``cassandra.load-policy.use-dc-aware``                        Set to ``true`` to use ``DCAwareRoundRobinPolicy``,
-                                                              defaults to ``false``.
+``cassandra.load-policy.use-dc-aware``                        Set to ``true`` if the load balancing policy requires a local
+                                                              datacenter, defaults to ``true``.
 
-``cassandra.load-policy.dc-aware.local-dc``                   The name of the local datacenter for ``DCAwareRoundRobinPolicy``.
+``cassandra.load-policy.dc-aware.local-dc``                   The name of the datacenter considered "local".
```

Comment:
Ideally we say this is required when use-dc-aware is true... but I am fine with the current status as well